src/index.ts

```diff
@@ -1,5 +1,21 @@
 #!/usr/bin/env node
+import { cosmiconfigSync }  from "cosmiconfig";
+import program  from "commander";
 
 import { run } from "./runner";
+import { ConfigInterface } from "./config.interface";
 
-run();
+const cliConfig = program
+    .option('-p, --project [project]', 'TS project configuration file (tsconfig.json)')
+    .parse(process.argv)
+
+const moduleName = 'ts-prune';
+const explorerSync = cosmiconfigSync(moduleName);
+const fileConfig = explorerSync.search();
```

Comment:
Since we're not really using the cosmicconfig configuration file, and not documenting the configurability, do you think it's worth adding this dependency and overhead here?

Reply:
Added `ignore` options, updated the docs.